cilium/cmd/helpers.go

```diff
@@ -79,3 +82,46 @@ func requireServiceID(cmd *cobra.Command, args []string) {
 		Usagef(cmd, "Empty service id argument")
 	}
 }
+
+//JSONOutput flag to enable json outout
+var JSONOutput bool
```

Comment:
Is there any way you could insert these two flags as command-specific config? It would allow for much easier testing of new output option.

Reply:
@nebril not sure if a specific flag will help. Testing is made using the bash scripts, and testing these commands need the cilium-agent running, something that we don't have with the go unittest. 

Have a particular flag per command implies the creation of 20 more variables that I'm not sure that will help.

I've just added the test for DumpJSON function, which works without the agent. 

If you still need the changes, ping me, and I'll make it.

Reply:
Thanks for adding the test, aside from CI not passing LGTM.